common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -414,6 +414,12 @@ div.problem {
     }
   }
 
+  div.inline {
+    > span {
```

Comment:
@frrrances Ah, I would love to. However, the inline class is already defined, and used in the sourcecode to specify when some elements should be inlined. Note that I am not defining a new class, but merely leveraging the presently defined one to do what it should be doing. See display.css (the file that this PR is changing), line 53:
<pre>  .inline {
    display: inline;
  }</pre>